variables.tf

```
@@ -194,6 +194,11 @@ variable "public_access_master_instance" {
   default     = false
 }
 
+variable "allocated_ip_range" {
+  description = "For MYSQL db, adding this property to Cloud SQL modules will allow users to select a specific allocated range for their private instances."
```

This is mysql_db module. So we know it's for MySQL.

```suggestion
  description = "Allow users to select a specific IP range for their private instances."
```

Also, what would be the recommended IP width for this? `/16`? `/20`? `/24`? `/32`.